src/main/java/com/google/sps/utils/QuizTimingPropertiesUtils.java

```diff
@@ -49,4 +78,44 @@ public Boolean userTookQuiz(String usersQuizTime, String currentQuizTime) {
         } 
         return false;
     }
+
+    //This function checks to see if the quiz is outdated
+     public Boolean newDayNewQuiz(Object current_quiz_time) {
+        String quiz_date;
+        try {
+            quiz_date = DateFormat.getDateInstance().format(current_quiz_time);
+        } catch(IllegalArgumentException e) {
+            log.severe("Given a null Parameter");
+            return null;
+        }
+ 
+        String today_date = DateFormat.getDateInstance().format(new Date());
+        if(today_date.compareTo(quiz_date) > 0) {
+            return true;
+        }
+        return false;
+    }
+    
+    //This function gets a new quiz question if the quiz is outdated
+    public Object getNewQuestion(Entity game_entity, DatastoreService datastore) {
```

Comment:
In my experience, `Object` is never a useful return type. Please make the return type more specific.